invenio_search_ui/assets/js/invenio_react_search_ui/Results.js

```diff
@@ -0,0 +1,146 @@
+// /*
+//  * This file is part of Invenio.
+//  * Copyright (C) 2020 CERN.
+//  *
+//  * Invenio is free software; you can redistribute it and/or modify it
+//  * under the terms of the MIT License; see LICENSE file for more details.
+//  */
+
+import React, { Component } from "react";
+import { Grid, Card, Image, Item, Label, Divider } from "semantic-ui-react";
+import _truncate from "lodash/truncate";
+import {
+  ActiveFilters,
+  Count,
+  LayoutSwitcher,
+  Pagination,
+  ResultsMultiLayout,
+  ResultsList,
+  ResultsGrid,
+  ResultsPerPage,
+  Sort
+} from "react-searchkit";
+
+const SpanWithMargin = ({ text, margin }) => {
+  const size = "0.5em";
+  let style;
+  switch (margin) {
+    case "left":
+      style = { marginLeft: size };
+      break;
+    case "right":
+      style = { marginRight: size };
+      break;
+    default:
+      style = { margin: `0 ${size}` };
+  }
+  return <span style={style}>{text}</span>;
+};
+
+export class Results extends Component {
+  constructor(props) {
+    super(props);
+
+    this.sortValues = this.props.sortValues;
+    this.resultsPerPageValues = this.props.resultsPerPageValues;
+  }
+
+  renderResultsListItem = (result, index) => {
+    const metadata = result.metadata;
+    return (
+      <Item key={index} href={`#`}>
+        <Item.Image
+          size="small"
+          src={result.imageSrc || "https://placehold.it/200"}
+        />
+        <Item.Content>
+          <Item.Header>{metadata.title}</Item.Header>
+          <Item.Description>
+            {_truncate(metadata.keywords.join(","), { length: 200 })}
```

Comment:
@lnielsen in `invenio-search-ui` we had a [marc21](https://github.com/inveniosoftware/invenio-search-ui/blob/master/invenio_search_ui/static/templates/invenio_search_ui/marc21/default.html) folder with some fields displayed. What would be a sensible default here to display records? I guess it is for sure overridden in `cookiecutter-invenio-instance`, but what should we have here?

Reply:
Everything related to marc21 can be removed.

Something like record.title + record.description